src/middlewared/middlewared/plugins/account.py

```diff
@@ -984,6 +984,8 @@ def do_update(self, app, audit_callback, pk, data):
         reset_tally(user['username'])
         self.middleware.call_sync('service.control', 'RELOAD', 'ssh').wait_sync(raise_error=True)
         self.middleware.call_sync('service.control', 'RELOAD', 'user').wait_sync(raise_error=True)
+        # NAS-136461: Possible group membership change. Reload nfs to avoid a 10 min delay.
+        self.middleware.call_sync('service.control', 'RELOAD', 'nfs').wait_sync(raise_error=True)
```

Comment:
>[2025/07/03 04:43:05] (ERROR) ServiceService.reload():403 - Service 'nfs' not running after reload

This will generate a log message on every user update. It also doesn't catch case where people change group membership via `group.update` of the `users` key.

Reply:
Second part of this is still an issue.